pages/doc/sources_managing.md

```diff
@@ -39,15 +55,15 @@ Filter sources by status.
 <tr>
 <td width="50%">
 <p>Filter sources by tag. In most environments, administrators add source tags. You can <a href="tags_overview.html#source-tags">add source tags</a> from the UI or programmatically.</p>
-<p>In the screenshot on the right, we've filtered by the tag wavefront.aws.billing and selected Recent Metrics (which shows as <strong>ok</strong> in the search bar).</p></td>
+<p>In the screenshot on the right, we've filtered by the tag `integration.linux`.</p></td>
 <td width="50%"><img src="/images/sources_tag_paths.png" alt="multiple tags for selection"></td>
 </tr>
 </tbody>
 </table>
 
 ### Step 2: Examine the Source
 
-When you select a source in the Sources browser, you can examinine it in more detail on a second page.
+When you select a source in the Sources browser, you can examine it in more details on a second page.
```

Comment:
When you select a source in the Sources browser, you can examine it in more detail on a second page.